enhancements/network/configure-ovs-alternative.md

```diff
@@ -254,6 +258,25 @@ actions:
   More details on this can be found in the enhancement:
   https://github.com/openshift/enhancements/pull/1525
 
+
+#### UPI
+
+In UPI clusters where the deployer has the ability to do arbitrary network
+configuration before deployment, there may be no need for NMState to be
+involved at deploy-time. Instead, the deployer can do the bridge
+configuration and then touch a file to indicate that configure-ovs should
```

Comment:
Not a great UX requiring the user to touch a file (via MCO?).
Maybe there is a way to tell when it is a UPI install right before nmstate executes and skip it?